### ISSUE TITLE
Snap SublimeMerge

### ISSUE THREAD

**Issue by 3v1n0** [snapcrafters/sublime-text#20]
Not really same project, but from same authors... 

https://www.sublimemerge.com

Same snapcrafters? :)

**Comment by ryanpcmcquen** [comment]
@snapcrafters, pretty please!
@snap-advocacy-bot

**Comment by popey** [comment]
I've built a snap of sublime-merge, just waiting on the previous registrant of the name, so we can transfer it to snapcrafters.

**Comment by ryanpcmcquen** [comment]
@popey, you rock! Is there any where we can track that name transfer?

**Comment by popey** [comment]
I have the name transferred, and the snap works fine as classic, but I'm trying to confine it.  Will update when I have it working.

**Comment by 3v1n0** [comment]
Lovely, although I'm not sure it could be confined considering that as any editor it should be able to access any repo... Like I'm doing many things in partitions or under dotted files.

In general would be nice to have something in between of classic and confined, that will allow to blacklist the access list, instead of white-list what we want.

**Comment by popey** [comment]
I felt similarly, and had it working as classic. But I have difficulty coming up with a concrete argument for it being classic. Can you articulate how you're using git repos on other partitions or dotted files which would fail under strict confinement?

**Comment by 3v1n0** [comment]
Those are mostly the reasons... I might need to manage git repos that I save somwhere else from my home, or I tend to use /tmp a lot for things I want to trash after some time, but in the mean time I clone them and forget.

In theory I could do a temp inside my home, but then i forge to delete it... :)
But probably I'm just using things weirdly :stuck_out_tongue:
